### PR TITLE
Channel detail mobile

### LINKED ISSUE #598: Implement Channel Details mobile screen
https://invis.io/2WNS9T68FX6

### PR DESCRIPTION
**Pull request by tanx**
Closes #598

### CODE REVIEW

File: src/view/channel-detail-mobile.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    flex: 1,
+    justifyContent: 'flex-end',
+  },
+  deleteBtn: {
+    marginTop: 15,
+    minHeight: 50,
+  },
+  deleteBtnText: {
+    color: color.pinkSig,
+    fontSize: 12,
```
Comment (valentinewallace) [inline src/view/channel-detail-mobile.js]:
```suggestion
     fontSize: font.sizeS,
```

File: src/view/channel-detail-mobile.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import Card from '../component/card';
+import { color } from '../component/style';
+
+const styles = StyleSheet.create({
+  btnWrapper: {
+    flex: 1,
+    justifyContent: 'flex-end',
+  },
+  deleteBtn: {
+    marginTop: 15,
```
Comment (valentinewallace) [inline src/view/channel-detail-mobile.js]:
```suggestion

```

File: src/view/channel-detail-mobile.js (hunk 1 above)
Comment (valentinewallace) [reply src/view/channel-detail-mobile.js]: + importing `font`

Comment (tanx) [reply src/view/channel-detail-mobile.js]: Nice catch. Fixed this also for the desktop version.

File: src/view/channel-detail-mobile.js (hunk 2 above)
Comment (tanx) [reply src/view/channel-detail-mobile.js]: We need the margin on smaller screens so the button isn't crammed against the fields when scrolling.

**Pull request merged**